### ISSUE TITLE
[BO] Trie avec les filtre des Membres

### ISSUE THREAD

**Issue by MaxLgrs** [yesiman/ekoal.web.ekoalitos.fo.ocp#238]
Quand je trie en cliquant des les cases du filtre rien ne se passe.
![image](https://user-images.githubusercontent.com/44435658/90611094-92df2480-e206-11ea-818a-790b827f82a3.png)

**Comment by MaxLgrs** [comment]
Cela fonctionne pour les projets

**Comment by SariTani** [comment]
cela no fonctionne pas pour moi :/ 

- les membres sont pas par ordre alphabétique

- le tri ne marche pas

**Comment by SariTani** [comment]
Manque juste le tri lorsque on coche les cases su menu à gauche

**Comment by yesiman** [comment]
Ba j'avais essayé cela marchait ! attend je reteste !

**Comment by yesiman** [comment]
Tu peux réessayer stp, et me dire la manipulation qui pose problème ? Merci

**Comment by SariTani** [comment]
Oui,  la par exemple je coche uniquement l'option "farmers" et j'ai tout les membres de la plateforme qui apparaissent.
![image](https://user-images.githubusercontent.com/44605319/94912057-5eb67f00-04a7-11eb-8296-e97dad5d5a7f.png)

**Comment by yesiman** [comment]
Oulala moi je n'ai que 5 résultats, comment tu fais, tu vas directement
dans le mode liste puis décoche members puis decoche tout sauf farmers ?
Y a t'il également une rechercher texte en plus de ta requête ?

**Comment by SariTani** [comment]
Non, pas de recherche texte juste en décrochant tout sauf "farmers"

**Comment by yesiman** [comment]
Alors désolé mais peux tu tester oubien sur un autre navigateur, ou en
vidant le cache ?
Chez moi (sur test.ocp.ekoal.org) cela fonctionne donc c'est très bizarre.
A plus tard merci

**Comment by SariTani** [comment]
J'ai testé avec autre navigateur (firefox que je n'utilise jamais) et cela ne fonctionne pas non plus ... Dis moi si t'as besoin d'autres tests

**Comment by yesiman** [comment]
D'accord non merci j'ai compris cela se produit uniquement quand on passe
par le menu "members" dans la barre de compteurs.
Et moi je testais toujours en passant par la zone de recherche avec un
texte vide.
Je corrige ça !
Merci

**Comment by yesiman** [comment]
En fait je cliquais sur l'onglet network qui ne générait pas le soucis...

**Comment by SariTani** [comment]
ah oui en effet! Super merci!

**Comment by yesiman** [comment]
Voila je met en ligne en test ça doit être ok maintenant ! Maxime aussi me l'avait remonté mais je comprenais pas ! Sorry
